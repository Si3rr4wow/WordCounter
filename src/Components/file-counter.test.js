import React from 'react'
import renderer from 'react-test-renderer'
import FileCounter from './file-counter'

it('Renders the file input', () => {
  const tree = renderer
    .create(<FileCounter/>)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
