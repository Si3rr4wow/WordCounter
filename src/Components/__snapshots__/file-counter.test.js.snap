// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the file input 1`] = `
<div
  className="card"
>
  <div
    className="card-header"
  >
    <h4>
      Use me if you've got a larger file to analyse.
    </h4>
  </div>
  <div
    className="card-body"
  >
    <p>
      I spin up web workers to count the words in the file you pass me. This means that the event loop doesn't get clogged up during the long running process.
    </p>
    <input
      className="form-control-file"
      disabled={false}
      onChange={[Function]}
      type="file"
    />
  </div>
</div>
`;
